src/components/templates/OrderTemplate.jsx

```diff
@@ -0,0 +1,201 @@
+        })
+      );
+      //item : 각 상품,
+    });
+
+    return renderComponent;
+  };
+
+  return (
+    <div className="py-20">
+      <div className="block mx-auto max-w-[1024px] w-[100%]">
+        <div className="border py-2 text-center">
+          <h1 className="text-sm font-bold">주문하기</h1>
+        </div>
+        <div className="border py-4">
+          <h2 className="test-sm font-bold">배송지 정보</h2>
+        </div>
+        <div className="border p-4">
+          <div className="flex items-center gap-4">
+            <span>홍길동</span>
+            <span className="text-blue-400 bg-blue-100 rounded-md text-xs p-1 ">
+              기본배송지
+            </span>
+          </div>
+        </div>
+        <InputGroup
+          label="전화번호"
+          className="border p-4"
+          placeholder="받으시는 분의 전화번호를 입력해주세요"
+        />
+        <InputGroup
+          label="주소명"
+          className="border p-4"
+          placeholder="받으시는 분의 주소명을 입력해주세요"
+        />
+        <div className="border py-4 font-bold">
+          <h2>주문상품 정보</h2>
+        </div>
+        {/* 각 주문의 정보 */}
+        <OrderItems />
+
+        {/* 총 주문 금액 */}
+        <div className="border p-4 flex item-center justify-between">
+          <h3 className="font-bold text-xl">총 주문 금액</h3>
+          <span className="price text-xl text-indigo-700">
+            {comma(totalPrice)}원
+          </span>
+        </div>
+        {/* 전체 동의, 구매조건 확인 및 결제 진행 동의 */}
+        <div className="flex flex-col p-4 gap-4">
+          <div className="flex items-start gap-2">
+            <input
+              type="checkbox"
+              id="all-agree"
+              ref={allAgreeRef}
+              checked={agreePayment && agreePolicy}
+              onChange={handleAllAgree}
+            />
+            <label htmlFor="all-agree" className="text-sm whitespace-nowrap">
+              전체 동의
+            </label>
+          </div>
+          <div className="flex items-start gap-2">
+            <input
+              type="checkbox"
+              id="agree"
+              name="payment-agree"
+              ref={agreePaymentRef}
+              checked={agreePayment}
+              onChange={handleAgreement}
+            />
+            <label htmlFor="agree" className="text-sm whitespace-nowrap">
+              구매조건 확인 및 결제 진행 동의
+            </label>
+          </div>
+          <div className="flex items-start gap-2">
+            <input
+              type="checkbox"
+              id="policy"
+              name="policy-agree"
+              ref={agreePolicyRef}
+              checked={agreePolicy}
+              onChange={handleAgreement}
+            />
+            <label htmlFor="agree" className="text-sm whitespace-nowrap">
+              개인정보 제 3자 제공동의
+            </label>
+          </div>
+          {/* 결제하기 버튼 */}
+          <button
+            onClick={() => {
+              // 동의가 이뤄지지 않았을 경우 처리
+              if (agreePayment === false || agreePolicy === false) {
```

Comment:
```suggestion
              if (!agreePayment || !agreePolicy) {
```

여기도 간단하게 ! 로 처리해주셔도 되겠습니다~!